src/core/src/pass/serialize.cpp

```diff
@@ -1064,26 +1058,25 @@ void serializeFunc(std::ostream& xml_file,
 }  // namespace
 
 namespace ov {
-bool pass::Serialize::run_on_model(const std::shared_ptr<ngraph::Function>& f_orig) {
+bool pass::Serialize::run_on_model(const std::shared_ptr<ov::Model>& model) {
     RUN_ON_FUNCTION_SCOPE(Serialize);
-    auto f = f_orig->clone();
```

Comment:
is this cloned was not needed at all? or you have changed the logic in the PR so the clone is not needed?
I suppose the answer is that clone is not needed at all

Reply:
I didn't change the logic. Let's wait for CI, I don't see that we need to have this clone

Reply:
Seehttps://github.com/openvinotoolkit/openvino/pull/9840

Reply:
Fixed original issue